diff --git a/node_modules/@expo/config-plugins/build/utils/plugin-resolver.js b/node_modules/@expo/config-plugins/build/utils/plugin-resolver.js
index 424ede4..06df7cb 100644
--- a/node_modules/@expo/config-plugins/build/utils/plugin-resolver.js
+++ b/node_modules/@expo/config-plugins/build/utils/plugin-resolver.js
@@ -148,7 +148,7 @@ function resolveConfigPluginExport({ plugin, pluginFile, pluginReference, isPlug
 exports.resolveConfigPluginExport = resolveConfigPluginExport;
 function requirePluginFile(filePath) {
     try {
-        return require(filePath);
+        return require(/* webpackIgnore: true */ filePath);
     }
     catch (error) {
         // TODO: Improve error messages
