diff --git a/node_modules/@expo/config/node_modules/@expo/config-plugins/build/utils/plugin-resolver.js b/node_modules/@expo/config/node_modules/@expo/config-plugins/build/utils/plugin-resolver.js
index 19fec1f..737058c 100644
--- a/node_modules/@expo/config/node_modules/@expo/config-plugins/build/utils/plugin-resolver.js
+++ b/node_modules/@expo/config/node_modules/@expo/config-plugins/build/utils/plugin-resolver.js
@@ -111,7 +111,7 @@ function resolveConfigPluginExport(result, configFile) {
 exports.resolveConfigPluginExport = resolveConfigPluginExport;
 function requirePluginFile(filePath) {
     try {
-        return require(filePath);
+        return require(/* webpackIgnore: true */ filePath);
     }
     catch (error) {
         // TODO: Improve error messages
