diff --git a/node_modules/xdl/build/Webpack.js b/node_modules/xdl/build/Webpack.js
index 16b8125..bb1f04b 100644
--- a/node_modules/xdl/build/Webpack.js
+++ b/node_modules/xdl/build/Webpack.js
@@ -535,8 +535,9 @@ async function getAvailablePortAsync(options) {
 }
 
 function setMode(mode) {
-  process.env.BABEL_ENV = mode;
-  process.env.NODE_ENV = mode;
+  // Webpack tries to resolve this, resulting in "production" = mode;
+  // process.env.BABEL_ENV = mode;
+  // process.env.NODE_ENV = mode;
 }
 
 function validateBoolOption(name, value, defaultValue) {
